app/Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM centos:8
+FROM alpine:3.15.3
```

Comment:
I think we should update the dq alpine image directly and then reference the image in this repo - in the future this would save us time so we only update one image

Reply:
The dq-alpine image your refering to is a different one. I can do it create a new dq-alpine one.